docs/howtos/basic-auth.md

```diff
@@ -1,24 +1,26 @@
 # Authentication
 
-Ambassador can authenticate incoming requests before routing them to a backing service. In this tutorial, we'll configure Ambassador to use an external third party authentication service. Note that if you're using the Ambassador Edge Stack, the [`External` filter](../../topics/using/filters) is a more powerful way to manage authentication. 
+IMPORTANT: This guide applies to the Ambassador API Gateway version of Ambassador, use of this guide on the Ambassador Edge Stack is not fully supported; use the [External Filter](../../topics/using/filters) instead.
```

Comment:
"Ambassador API Gateway version of Ambassador" reads awkwardly. Instead, I would say this:

IMPORTANT: This guide applies to the Ambassador API Gateway. Use of this guide with the Ambassador Edge Stack is not fully supported; use ...

Reply:
Fixed